src/cmd/policy.go

```diff
@@ -84,6 +93,45 @@ func RegoFuncReadFile(ctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
 	return nil, nil
 }
 
+func RegoFuncGetGithubRepo(ctx rego.BuiltinContext, a, b *ast.Term) (*ast.Term, error) {
+
+	var org, repo string
+	if err := ast.As(a.Value, &org); err != nil {
+		return nil, err
+	} else if ast.As(b.Value, &repo); err != nil {
+		return nil, err
+	}
+
+	if org == "" {
+		log.Error().Msgf("Please provide a valid org")
+		return nil, nil
+	}
+
+	if repo == "" {
+		log.Error().Msgf("Please provide a valid repo")
+		return nil, nil
+	}
+
+	githubToken := viper.GetString("github-token")
+	authorizationHeader := fmt.Sprintf("token %s", githubToken)
+	githubAPIUrl := fmt.Sprintf("https://api.github.com/repos/%v/%v", org, repo)
+
+	response, err := getClientRest().R().
+		SetHeader("Accept", "application/vnd.github+json").
+		SetHeader("Authorization", authorizationHeader).
+		Get(githubAPIUrl)
+	cobra.CheckErr(err)
+
+	if response.IsError() == true {
+		log.Error().Msgf("%d: %s", response.StatusCode(), response)
```

Comment:
while we are on error messages - we could improve this one to call out it was an api call to github and this is the API response.

maybe something like

```
log.Error().Msgf("error requesting github repo metadata. CODE: %d REASON: %s", response.StatusCode(), response)
```